script/build-icons

```diff
@@ -1,11 +1,28 @@
 #!/bin/sh
 
-dir=$(cd $(dirname $0) && cd "../src/common/icons" && pwd)
-src="${dir}/icon.svg"
+# script/build-icons: Build optimized png icons in all required sizes and
+#                     colors from the svg icons.
+#
+#                     Running this script requires `inkscape` and `pngcrush`
+#                     executables in your PATH.
```

Comment:
The indentation seems a bit excessive 😁 

```suggestion
# Build optimized PNGs in all required sizes and colors from the SVG icons.
#
# Requires `inkscape` and `pngcrush` executables in your PATH.
```

Reply:
🤷 I just went with how they format scripts in [scripts-to-rule-them-all](https://github.com/github/scripts-to-rule-them-all)